src/components/sketches/test4.ts

```diff
@@ -0,0 +1,110 @@
+import p5 from 'p5';
+export default function sketch(p: p5): void {
+  let canvas: p5.Renderer;
+
+  const particles: Particle[] = [];
+  const windowWidth = window.innerWidth;
+  const windowHeight = window.innerHeight;
+
+  p.setup = () => {
+    canvas = p.createCanvas(window.innerWidth, window.innerHeight);
+    canvas.id('p5-background');
+
+    const particlesLength = Math.min(Math.floor(window.innerWidth / 10), 200); //note this line
```

Comment:
what is this comment for?

Reply:
also.. shouldnt this be called `numParticles`

Reply:
@forrestburton do you know about this? Seems like an easy fix though. I like @BryanPan342  suggestion a lot better anyway in naming conventions for this issue.